recipes/hkl/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "hkl" %}
+{% set version = "5.0.0.2173" %}
+{% set sha256 = "5f3991d2af4ff01d6f6e29d0176af9bc1b54e6a8598046a050ad804fedd860f2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/picca/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - no-docs.patch
+
+build:
+  number: 0
+  skip: True  # [py<36 or win or osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - autoconf
+    - automake
+    - libtool
+    - m4
+    - pkg-config
+  host:
+    - glib
+    - gobject-introspection
+    - gsl
+    - pygobject
+    - python
+  run:
+    - python
+    - gsl
+    - libiconv
+    - libopenblas
```

Comment:
See docs on BLAS on how to correctly do this.

Reply:
What do you mean here? I don't think the package depends on `libopenblas`, I just wanted to get rid of the warnings in the packing step.

Reply:
@isuruf, I'd like to finish with this PR, but didn't quite get your points. Could you please clarify how it should be done? I was trying to get rid of the warnings in the build logs, which actually didn't break the builds. Any suggestions are much appreciated. Thank you!

Reply:
See https://conda-forge.org/docs/maintainer/knowledge_base.html#blas

Reply:
That's an informative link, thanks! I saw you pushed a commit https://github.com/conda-forge/staged-recipes/pull/10819/commits/13952894632011dafad0be6ecfbaf53eb53844a7, so I assume this is resolved.